clang/docs/ReleaseNotes.rst

```diff
@@ -181,6 +181,9 @@ Bug Fixes to C++ Support
   and (`#79745 <https://github.com/llvm/llvm-project/issues/79745>`_)
 - Fix incorrect code generation caused by the object argument of ``static operator()`` and ``static operator[]`` calls not being evaluated.
   Fixes (`#67976 <https://github.com/llvm/llvm-project/issues/67976>`_)
+- Fix incorrect merging of modules which contain using declarations which shadow
+  other declarations. This could manifest as ODR checker false positives.
+  Fixes (`#TODO <https://github.com/llvm/llvm-project/issues/TODO>`_)
```

Comment:
Oh, if it is the case, we should mark this as draft. Otherwise, if I don't look into every line and I see your funcitonal change is pretty simple, it'll be bad if I merge it directly...

Reply:
We usually do small fixups like this during review / post approval, this will not be merged like this, don't worry.
People on the LLVM community tend to be quite flexible on these things.

Reply:
Let's file a real issue.

Reply:
> We usually do small fixups like this during review / post approval, this will not be merged like this, don't worry.
People on the LLVM community tend to be quite flexible on these things.

Yeah, but we still need to a real issue in the end of the day, right?

Reply:
(I don't know why the comment got duplicated)

Reply:
For example, I need to what's the behavior before the patch, then I can have a better feeling about how should we test it.

Reply:
The MR is split in two commits, the first one adds the test, if you click on the last commit, you can see the change in the expectation.